mobetta/models.py

```
@@ -53,7 +53,16 @@ def get_statistics(self):
         - fuzzy messages
         - obsolete messages
         """
-        pofile = self.get_polib_object()
+        try:
+            pofile = self.get_polib_object()
+        except:
```

I'd prefer at least a `logging` call here instead of just silently ignoring things.